src/main/java/org/elasticsearch/gateway/TransportNodesListGatewayStartedShards.java

```diff
@@ -59,8 +62,8 @@ public TransportNodesListGatewayStartedShards(Settings settings, ClusterName clu
         this.nodeEnv = env;
     }
 
-    public ActionFuture<NodesGatewayStartedShards> list(ShardId shardId, String[] nodesIds, @Nullable TimeValue timeout) {
-        return execute(new Request(shardId, nodesIds).timeout(timeout));
+    public ActionFuture<NodesGatewayStartedShards> list(ShardId shardId, String indexUUID, String[] nodesIds, @Nullable TimeValue timeout) {
+        return execute(new Request(shardId, indexUUID, Arrays.asList(nodesIds)).timeout(timeout));
```

Comment:
yeah so I did this since I didn't want the `String, String...` signature.. kind of confusing